packages/shorebird_cli/lib/src/cache.dart

```diff
@@ -218,7 +217,7 @@ class AotToolsArtifact extends CachedArtifact {
 
   @override
   String get storageUrl =>
-      '${cache.storageBaseUrl}/${cache.storageBucket}/shorebird/${shorebirdEnv.shorebirdEngineRevision}/aot-tools.dill';
+      '${cache.storageBaseUrl}/${cache.storageBucket}/shorebird/${shorebirdEnv.shorebirdEngineRevision}/$name';
```

Comment:
I think this will still fail when we are using flutter revisions that pre-date the dill.

Reply:
Fixing.